src/scripts/components/CRCMaker.js

```diff
@@ -254,14 +264,16 @@ export default class CRCMaker extends Component {
                   </Dialog>
                 }
 
+                <button onClick={this.toggleImport}>Import</button>
+
                 <button onClick={this.toggleExport}>Export</button>
                 { state.exportVisible &&
                   <Dialog title='Export JSON' onClose={this.toggleExport}>
-                    <pre id='text-export' className='syntax'
-                      dangerouslySetInnerHTML={{__html: syntaxHighlight(state.cards)}} />
-
                     <button className='copy' data-clipboard-target='#text-export'>Copy</button>
                     <button onClick={this.toggleExport}>Close</button>
+
+                    <pre id='text-export' className='syntax'
```

Comment:
Why was this moved?

Reply:
Oh, sorry. That was another idea I had that I forgot I left in (I didn't like having to scroll down to select "copy".

I can remove that from the branch.